pennylane/grouping/utils.py

```diff
@@ -523,7 +523,7 @@ def pauli_word_to_matrix(pauli_word, wire_map=None):
     for wire_label, wire_idx in wire_map.items():
         if wire_label in pauli_word.wires.labels:
             op_idx = pauli_word.wires.labels.index(wire_label)
-            pauli_mats[wire_idx] = getattr(qml, pauli_names[op_idx]).matrix
+            pauli_mats[wire_idx] = getattr(qml, pauli_names[op_idx]).compute_matrix()
```

Comment:
Doesn't `compute_matrix` need parameters?

Reply:
Not for Paulis, so here we are ok. But maybe you're right, it may not me safe. I am not entirely sure what to do here.

Reply:
I added a comment, and unless someone changes this function it should be safe!